### PR TITLE
US-3.5: Update events portlet display

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/vue-app/events/components/EventsApp.vue [hunk 1]
```diff
@@ -91,29 +95,31 @@
...
-                sm2>
+                sm1>
                 <v-layout column>
-                  <v-avatar size="36">
+                  <v-avatar size="36" class="pa-6">
                     <span class="headline" style="color: #568DC9">20</span>
                   </v-avatar>
                   <v-avatar size="20">
-                    <span class="ml-4" style="font-size: 15px;color: #568DC9">OCT</span>
+                    <span class="ml-6" style="font-size: 15px;color: #568DC9;height: 35px">OCT</span>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/events/components/EventsApp.vue]: We should avoid style here

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/events/components/EventsApp.vue]: Ok

File: digital-workplace-webapps/src/main/webapp/vue-app/events/components/EventsApp.vue [hunk 2]
```diff
@@ -125,4 +131,7 @@
           </v-layout>
     </v-flex></v-layout></v-container>
   </v-app>
-</template>
\ No newline at end of file
+</template>
+<style>
+  .blueText{font-size: 15px;color: #568DC9}
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/events/components/EventsApp.vue]: To be moved into global css file for the moment

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/events/components/EventsApp.vue]: Done

**Pull request merged**